### PR TITLE
[feature] new search

### PR DESCRIPTION
**Pull request by ponsfrilus**
- add EPNET
- add actu.epfl.ch
- add missing icones from https://ponsfrilus.github.io/searchplugins/

### CODE REVIEW

**Review by obieler — COMMENTED** [review]
Would "EPFL Directory" or "EPFL IT Directory" be more correct?

**Comment by ponsfrilus** [comment]
Instead of EPNET?

**Comment by obieler** [comment]
> Instead of EPNET?

Yes. EPNET, which stands for EPFL Network, is broader than the directory.

**Comment by ponsfrilus** [comment]
> > Instead of EPNET?
> 
> Yes. EPNET, which stands for EPFL Network, is broader than the directory.

Here you go (force pushed and reordered commits).

File: src/public/js/search.js [hunk 1]
```diff
@@ -63,6 +74,12 @@ $(function () {
         case 'go':
           setSearchOn('EPFL Go', arrayOfTerms[1]);
           break;
+        case 'epnet':
```
Comment (obieler) [inline src/public/js/search.js:78]: Can please you add a `case 'dir':`?

Comment (ponsfrilus) [reply src/public/js/search.js:78]: Sure. But please do your comments all at once, it's annoying to change it while re-submitted.

**Review by obieler — APPROVED** [review]

**Pull request merged**